package.json

```diff
@@ -41,9 +41,14 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "npm run prettier"
+      "pre-commit": "lint-staged"
     }
   },
+  "lint-staged": {
+    "./{server,src}/**/*.*": [
```

Comment:
can we also remove server/src directories in case we add more later?

e.g. there are dev scripts in dev that could also benefit from formatting, and new directories might pop up.

Reply:
sure